docs/reference/installation.rst

```diff
@@ -1,157 +1,104 @@
-                        mappings:
-                            SonataMediaBundle: ~
-
-If you picked Doctrine PHPCR:
-
-.. configuration-block::
-
-    .. code-block:: yaml
-
-        # config/packages/doctrine_phpcr.yaml
-
-        doctrine_phpcr:
-            odm:
-                auto_mapping: true
-                mappings:
-                    SonataMediaBundle:
-                        prefix: Sonata\MediaBundle\PHPCR
-
-Once you have done that, you can configure the Media bundle itself:
-
-.. configuration-block::
-
-    .. code-block:: yaml
-
-        # config/packages/sonata_media.yaml
-
-        sonata_media:
-            # if you don't use default namespace configuration
-            #class:
-            #    media: MyVendor\MediaBundle\Entity\Media
-            #    gallery: MyVendor\MediaBundle\Entity\Gallery
-            #    gallery_has_media: MyVendor\MediaBundle\Entity\GalleryHasMedia
-            db_driver: doctrine_orm # or doctrine_mongodb, doctrine_phpcr it is mandatory to choose one here
-            default_context: default # you need to set a context
-            contexts:
-                default:  # the default context is mandatory
-                    providers:
-                        - sonata.media.provider.dailymotion
-                        - sonata.media.provider.youtube
-                        - sonata.media.provider.image
-                        - sonata.media.provider.file
-                        - sonata.media.provider.vimeo
-
-                    formats:
-                        small: { width: 100 , quality: 70}
-                        big:   { width: 500 , quality: 70}
-
-            cdn:
-                server:
-                    path: /uploads/media # http://media.sonata-project.org/
-
-            filesystem:
-                local:
-                    directory:  '%kernel.root_dir%/../public/uploads/media'
-                    create:     false
+Configuration
+=============
+
+SonataMediaBundle Configuration
+-------------------------------
+
+.. code-block:: yaml
+
+    # config/packages/sonata_media.yaml
+
+    sonata_media:
+        class:
+            media: App\Entity\SonataMediaMedia
+            gallery: App\Entity\SonataMediaGallery
+            gallery_has_media: App\Entity\SonataMediaGalleryHasMedia
+        db_driver: doctrine_orm # or doctrine_mongodb, doctrine_phpcr it is mandatory to choose one here
+        default_context: default # you need to set a context
+        contexts:
+            default:  # the default context is mandatory
+                providers:
+                    - sonata.media.provider.dailymotion
+                    - sonata.media.provider.youtube
+                    - sonata.media.provider.image
+                    - sonata.media.provider.file
+                    - sonata.media.provider.vimeo
+
+                formats:
+                    small: { width: 100 , quality: 70}
+                    big: { width: 500 , quality: 70}
+
```

Comment:
If the previous nodes of the same level haven't extra LFs in between, I think we should not add them here either.
```suggestion
```